content/rs/installing-upgrading/configuring/cluster-lba-setup.md

```diff
@@ -0,0 +1,100 @@
+---
+Title: Setting Up a Cluster Behind a Load Balancer
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When you want to setup a Redis Enterprise cluster in an environment that doesn't allow DNS, you can use a load balancer agent (LBA to direct traffic between the cluster nodes.
```

Comment:
We need to define "load balancer agent". As written, it's unclear whether this agent is a feature of RS or some external agent.

Reply:
It should be 'Load Balancer Appliance" not "agent". Another option is just living the A out and just using LB (Load Balancer) instead, as it can be more generic (LB can be virtual and does not have to be deployed as Appliance).